src/app/app/dashboard/page.tsx

```diff
@@ -1,4 +1,34 @@
+import Accordion from "@/components/Accordion";
+
 export default function Page() {
+  //Need to make the data an separated file.
```

Comment:
Any suggestion for the replacement of this kind of data?

Reply:
We can move this to our "settings.tsx" file inside the "shared" folder.

Also, when leaving comments like that, remember of marking them as `TODO`, as it gets highlighted by most IDEs and text editors:
```suggestion
  // TODO: Need to make the data an separated file.
```